webapp/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:22.04
+
+LABEL "org.opencontainers.image.source"="https://github.com/get10101/10101"
+LABEL "org.opencontainers.image.authors"="hello@10101.finance"
+
+ARG BINARY=target/release/webapp
+
+RUN apt-get update && \
+    apt-get install libpq5 -y && \
+    apt-get install sqlite3 -y && \
+    apt-get install ca-certificates openssl -y
+
+
+RUN mkdir cert
+
+RUN cd cert && \
+    openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 3650 -nodes -subj "/C=XX/ST=StateName/L=CityName/O=CompanyName/OU=CompanySectionName/CN=CommonNameOrHostname"
```

Comment:
nit: we might want to to adapt the details just so that it doesn't look too generic?

Reply:
I honestly don't know what to adapt here. I just took what I received from you 🙈

Reply:
I honestly don't know what to adapt here. I just took what I received from you 🙈